proposals/incubation/istio.md

```diff
@@ -0,0 +1,375 @@
+### Name of the project
+
+Istio
+
+### Description
+
+Istio is an open source service mesh that transparently provides a uniform and efficient way to secure, connect, and monitor services in cloud native applications. It provides zero-trust networking, policy enforcement, traffic management, load balancing, and monitoring, without requiring applications to be rewritten.
+
+Features include:
+
+* Secure service-to-service communication based on zero-trust principles, using TLS encryption and strong identity-based authentication and authorization
+* Automatic load balancing for HTTP, gRPC, WebSocket, and TCP traffic
+* Fine-grained control of traffic behavior with rich routing rules, retries, failovers, and fault injection
+* A policy API supporting access controls, rate limits and quotas, that can be connected to external authorization systems
+* Automatic generation of metrics, logs, and traces for all traffic within a cluster, including cluster ingress and egress
+* A data plane platform allowing for extension and customization of proxy behavior
+
+Istio is designed for extensibility, and can handle a diverse range of deployment needs. Istio’s control plane runs on Kubernetes, and is capable of managing workloads running in the same Kubernetes cluster, other Kubernetes clusters, and even workloads running outside Kubernetes on VMs.
+
+### Sponsor / Advisor from TOC
+
+To be appointed.
+
+### License
+
+Istio is licensed under the Apache 2.0 license.
+
+### Source control repositories
+
+* https://github.com/istio (everything in the org)
+* https://github.com/istio-ecosystem (everything in the org)
+
+### External dependencies
+
+Istio has external dependencies licensed under the following CNCF-approved licenses:
+
+* Apache 2.0
+* BSD License
+* ISC License
+* MIT License
+
+Five modules are licensed under the Mozilla Public License, and all have been [granted exceptions](https://github.com/cncf/foundation/blob/main/license-exceptions/cncf-exceptions-2019-11-01.spdx) by the CNCF governing board.
+
+Licenses for most Istio dependencies are automatically included at https://github.com/istio/istio/tree/master/licenses. A snapshot of modules and their corresponding licenses [can be found below](#licenses).
+
+Projects in istio-ecosystem will be brought in line with CNCF guidelines before submission.
```

Comment:
Do these subprojects not have a choice? Presuming they have a choice, isn’t it odd that integrated projects could become sandbox projects without a separate application and oversight?

Reply:
There are a types of project in istio-ecosystem: 

* contributed projects (e.g. Admiral)
* useful samples, maintained by the Istio project (e.g. wasm-extensions)
* things that are wildly out of date, which will be archived. 

For the first two, we will either help the teams move to their own orgs, or if they wish to remain in the "Istio incubator" (similar to the kubernetes-sigs repository, where projects exist but are not granted the same status as the parent project), they will individually need to be updated to meet CNCF standards.